### DISCUSSION TITLE
打包及部署问题

### DISCUSSION THREAD

**Discussion by frankelyns** [halo-dev/halo#4225] [Q&A]
### What is version of Halo has the issue?

release-2.7

### What database are you using?

MySQL 8.x

### What is your deployment method?

Fat Jar

### What happened?

console前端项目打包后找不到dist文件夹, 使用的打包命令：pnpm build， 在官方文档中未找到jar包形式部署文档。

**Comment by ruibaby** [comment]
开发环境运行和构建请参考：https://docs.halo.run/category/%E7%B3%BB%E7%BB%9F%E5%BC%80%E5%8F%91

此问题不属于 issue，我将转移到 Discussion。
